assets/src/apps/authoring/components/PropertyEditor/custom/SpokeCompletedOption.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { selectAllActivities } from 'apps/delivery/store/features/activities/slice';
+
+interface Props {
+  id: string;
+  value: number;
+  onChange: (value: number) => void;
+}
+
+export const SpokeCompletedOption: React.FC<Props> = ({ id, value, onChange }) => {
```

Comment:
This is a custom property that is needed to display the screen list in a drop down on property editor.